tools/branch_comparisons.md

```diff
@@ -0,0 +1,93 @@
+# Branch comparisons
+
+These links take to comparisons between branches.
+
+When it says "There isn’t anything to compare," this means no pull requests
+need to be forward-ported.
+
+Otherwise, those branches could use a forward-port pull request, see
+instructions on the
+[contributing guide](https://ignitionrobotics.org/docs/all/contributing#process).
+
+* **B**: Blueprint
+* **C**: Citadel
+* **D**: Dome
+* **main**: main branch
+
+Library | B ➡️  C | C ➡️  D | ➡️  main
```

Comment:
```suggestion
Library | B ➡️  C | C ➡️  D | D ➡️  main
```
When I looked at this table originally, it wasn't super clear to me what version of each repository was being compared to `main` until I clicked on the `➡️  main` link. It looks like the `➡️  main` column is comparing the most recent ignition release (in this case, `Dome`) to `main`, which makes sense - perhaps we should explicitly state this in the table so it's clear to all readers?

Reply:
Good point, I don't remember exactly what my reasoning was to left that out but I think that makes sense. Changed in 824176b170a8353a14f69bde100af7982fd8aa53